### DISCUSSION TITLE
confirmation  popup

### DISCUSSION THREAD

**Discussion by tiger360** [espanso/espanso#1243] [Feature requests and Ideas]
Hey,

I am wondering if there is a way get a confirmation popup.
This enables you to set up a confirmation pop-up before a script is run for example. This Setting comes in handy to protect against users accidentally running destructive operations.

Merci

**Comment by federico-terzi** [comment]
Hey @tiger360,

Thank you for the feedback! This sounds like a useful feature, I'll add it to the roadmap :)

Cheers :)
